src/ripple/protocol/Protocol.h

```diff
@@ -95,6 +95,8 @@ using LedgerIndex = std::uint32_t;
 */
 using TxID = uint256;
 
+std::size_t constexpr maxDeletableAMMTrustLines = 1500;
```

Comment:
Too high imho. 512?

Reply:
It takes 14-17 milliseconds to delete 1,500 trustlines and the metadata size is about 700K. So 1,500 seems acceptable.

Reply:
I changed to 512. AMMDelete unit-tests take too long with 1,500. 512 is good enough to cover majority of cases.